File: 1217. Minimum Cost to Move Chips to The Same Position/solution.py
class Solution(object):
  def minCostToMoveChips(self, position):
    """
    :type position: List[int]
    :rtype: int
    """
    even = 0
    odd = 0
    for chips in position:
      if chips % 2 == 0:
        even += 1
      else:
        odd += 1
    
    result = even
    if even > odd:
      result = odd
    
    return result

File: 1217. Minimum Cost to Move Chips to The Same Position/solution.cpp
class Solution {
public:
  int minCostToMoveChips(vector<int>& position) {
    int even = 0;
    int odd = 0;
    for (int index = 0; index < position.size(); index++){
      int chips = position[index];
      if (chips % 2 == 0){
        even += 1;
      }
      else{
        odd += 1;
      }
    }

    int result = even;
    if (even > odd){
      result = odd;
    }

    return result;
  }
};